src/app/forms/components/text-input/text-input.component.html

```diff
@@ -6,7 +6,7 @@
 
   <div *ngIf="hint" id="{{ name }}-hint" class="govuk-hint">{{ hint }}</div>
 
-  <app-field-error-message [error]="error ? error : warning" [name]="name"></app-field-error-message>
+  <app-field-error-message [error]="error" [name]="name"></app-field-error-message>
   <app-field-error-message [error]="warning" [name]="name"></app-field-error-message>
```

Comment:
would it make more sense to keep the top one and remove the bottom two? or does the ternary operator not evaluate how we'd expect?

Reply:
Either works, but I made it match all the other input components

Reply:
If you're matching the other input components, the last one needs to be an app-field-warning-message, not error